src/nerve/frontends/tui/commander/persistence.py

```
@@ -0,0 +1,277 @@
+            args = [f"id={repr(node_id)}", "session=session", f"command={repr(command)}"]
+            if cwd:
+                args.append(f"cwd={repr(cwd)}")
+
+            lines.append(f"await {node_class}.create({', '.join(args)})")
+            lines.append(f"print({repr(f'Created node: {node_id}')})")
+
+    # Recreate graphs (basic - no steps for now)
+    for graph in entities.get("graphs", []):
+        graph_id = graph["id"]
+        lines.append("")
+        lines.append(f"Graph(id={repr(graph_id)}, session=session)")
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)